src/components/Input/BaseInput/BaseInput.scss

```
@@ -12,11 +12,16 @@ $min-width: 40px;
     width: 100%;
     min-width: $min-width;
     min-height: $default-size;
+    background-color: white;
```

It's better to use color variable even for simple color to be sure to be consitant
```suggestion
    background-color: var(--color-white);
```